md/maintenance.md

```diff
@@ -0,0 +1,41 @@
+# Bonita Maintenance
+
+Sometimes the overall performances of the bonita platform could be altered cause by the different issue like  archive table becomes to be heavy.
```

Comment:
There is no identified altered performance issue. Only database size is big and the step to migrate to 7.7 is long.
To be rephrased to something that explains what we will learn in this page ( e.g. learn to do maintenance operation to reduce database size)